### PR TITLE
Mikecore

### PR DESCRIPTION
**Pull request by ecomodeller**
Progress so far:
Tests 98 failed 👎 , 180 passed 👍 

Pfs, Mesh, Res1d, xns11 not included 😢

### CODE REVIEW

**Comment by ecomodeller** [comment]
68 failed, 210 passed,

**Comment by ecomodeller** [comment]
57 failed, 249 passed,

**Comment by jsmariegaard** [comment]
9 failed, 297 passed

**Comment by jsmariegaard** [comment]
303 passed, 0 failed

File: .github/workflows/full_test.yml [hunk 1]
```diff
@@ -15,7 +15,8 @@ jobs:
     runs-on: windows-latest
```
Comment (jsmariegaard) [inline .github/workflows/full_test.yml:15]: Should this not be removed?

**Comment by ecomodeller** [comment]
It runs... :rocket: :1st_place_medal: :computer: :sweat_smile:

**Pull request merged**